### DISCUSSION TITLE
Gen:Variant.Lazy.577051

### DISCUSSION THREAD

**Discussion by Brayc1** [JPersson77/LGTVCompanion#235] [Q&A]
Hello, I've been using the tool for a while and love it but it has started flagging in bitdefender and virustotal as Gen:Variant.Lazy.577051 malicious software, any idea what's causing this?

**Comment by JPersson77** [comment]
Hi, please see this issue: https://github.com/JPersson77/LGTVCompanion/issues/234

Tl;dr it is a false positive. Please report it to bitdefe der

**Reply by Brayc1** [reply]
Thanks
